### PR TITLE
changes for proxy server

### PR DESCRIPTION
**Pull request by shabbyjoon**

### CODE REVIEW

**Comment by shabbyjoon** [comment]
@davydhong can you please review changes please? Thanks!

File: public/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 </head>
 <body>
-    <div id='app'></div>
+    <div id='raapp'></div>
```
Comment (davydhong) [inline public/index.html]: i think related-artists-app is more descriptive

Comment (davydhong) [inline public/index.html]: suggest   <div id='related-artists-app'></div>
just to be more descriptive

Comment (davydhong) [inline public/index.html]: i think related-artists-app is a better name. I think i need to change mine also.

File: public/index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
     <link rel="stylesheet" href="styles.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```
Comment (davydhong) [inline public/index.html]: not critical but you could try this also : 
https://getbootstrap.com/docs/3.3/getting-started/

**Pull request merged**